compose/cli/command.py

```diff
@@ -100,20 +107,12 @@ def get_config_path(self, file_path=None):
         if file_path:
             return os.path.join(self.base_dir, file_path)
 
-        supported_filenames = [
-            'docker-compose.yml',
-            'docker-compose.yaml',
-            'fig.yml',
-            'fig.yaml',
-        ]
-
-        def expand(filename):
-            return os.path.join(self.base_dir, filename)
+        filenames = SUPPORTED_FILENAMES
 
-        candidates = [filename for filename in supported_filenames if os.path.exists(expand(filename))]
+        (candidates, path) = find_candidates_in_parent_dirs(filenames, self.base_dir)
 
         if len(candidates) == 0:
-            raise errors.ComposeFileNotFound(supported_filenames)
+            raise errors.ComposeFileNotFound(filenames)
```

Comment:
Nitpick: we can just use SUPPORTED_FILENAMES instead of a local variable here and it'll be clearer (otherwise it looks like we're going to modify it)